misk/src/test/kotlin/misk/web/resource/StaticResourceActionTest.kt

```diff
@@ -211,34 +210,28 @@ class StaticResourceActionTest {
 
   class TestModule : KAbstractModule() {
     override fun configure() {
+      install(HttpClientsConfigModule(HttpClientsConfig(
```

Comment:
I'm not as experienced in Guice but would having this provided (as before) be faster in tests than being installed?

Reply:
What do you mean faster? Like how quickly the test runs? It'll be the same

Reply:
Yeah test run time. Sounds good 👍